scripts/env.sh

```diff
@@ -4,10 +4,13 @@ cd "$(cd -P -- "$(dirname -- "$0")" && pwd -P)" || exit 2
 
 # We dont care about return code here
 # shellcheck disable=SC2155
```

Comment:
Using `$PWD` instead of `$(pwd)` will get rid of the ShellCheck warning and the need for the exception in comment.

Reply:
Thanks for this. I have corrected it in env.sh